packages/loot-core/src/server/main.js

```diff
@@ -1265,7 +1270,7 @@ handlers['nordigen-get-banks'] = async function (country) {
 
   return post(
     getServer().NORDIGEN_SERVER + '/get-banks',
-    { country },
+    { country, showDemo: isDevelopmentEnvironment() || isPreviewEnvironment() },
```

Comment:
Looking at the code, everywhere we call either of these functions we do `isDevelopmentEnvironment() || isPreviewEnvironment()`. Do you think it would make sense to expose a new function (`exposeDebuggingFeatures()`?) that would make it simpler to check this condition?

Reply:
Good idea: https://github.com/actualbudget/actual/pull/790/commits/bfc1f6bd9bd132e3a654afb0590b7fc50f6f8a95